From 8e9103deda6d71221ef57789c289c8589bcabf12 Mon Sep 17 00:00:00 2001
From: althafvly <althafvly@gmail.com>
Date: Thu, 20 Jul 2023 10:11:53 +0000
Subject: [PATCH] Fix another build issue

../../storage/browser/quota/quota_device_info_helper.cc:16:69: error: implicit conversion from 'unsigned long long' to 'const size_t' (aka 'const unsigned int') changes value from 8589934592 to 0 [-Werror,-Wconstant-conversion]
constexpr size_t approximated_device_disk_size = 8ull * 1024 * 1024 * 1024;
---
 storage/browser/blob/blob_memory_controller.cc    | 2 +-
 storage/browser/quota/quota_device_info_helper.cc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/storage/browser/blob/blob_memory_controller.cc b/storage/browser/blob/blob_memory_controller.cc
index ec4f55358b2c5..2b4492212c6c3 100644
--- a/storage/browser/blob/blob_memory_controller.cc
+++ b/storage/browser/blob/blob_memory_controller.cc
@@ -52,7 +52,7 @@ using DiskSpaceFuncPtr = BlobMemoryController::DiskSpaceFuncPtr;
 // static
 #if BUILDFLAG(IS_ANDROID)
 float approximated_device_memory_gb_ = 4.0;
-constexpr size_t approximated_device_disk_size = 8ull * 1024 * 1024 * 1024;
+constexpr size_t approximated_device_disk_size = static_cast<size_t>(8ull * 1024 * 1024 * 1024);
 #else
 float approximated_device_memory_gb_ = 8.0;
 constexpr size_t approximated_device_disk_size = 200ull * 1024 * 1024 * 1024;
diff --git a/storage/browser/quota/quota_device_info_helper.cc b/storage/browser/quota/quota_device_info_helper.cc
index c67296bafa86e..8337289b76ca6 100644
--- a/storage/browser/quota/quota_device_info_helper.cc
+++ b/storage/browser/quota/quota_device_info_helper.cc
@@ -13,7 +13,7 @@ namespace {
 // static
 #if BUILDFLAG(IS_ANDROID)
 float approximated_device_memory_gb_ = 4.0;
-constexpr size_t approximated_device_disk_size = 8ull * 1024 * 1024 * 1024;
+constexpr size_t approximated_device_disk_size = static_cast<size_t>(8ull * 1024 * 1024 * 1024);
 #else
 float approximated_device_memory_gb_ = 8.0;
 constexpr size_t approximated_device_disk_size = 200ull * 1024 * 1024 * 1024;
-- 
2.25.1

